Detections/SecurityEvent/CredentialDumpingToolsFileArtifacts.yaml

```diff
@@ -0,0 +1,38 @@
+id: 32ffb19e-8ed8-40ed-87a0-1adb4746b7c4
+name: Credential Dumping Tools - File Artifacts
+description: |
+   'This query detects the creation of credential dumping tools files. Several credential dumping tools export files with hardcoded file names.
+   Ref: https://jpcertcc.github.io/ToolAnalysisResultSheet/'
+severity: High
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1003.001
+query: |
+  // Enter a reference list of malicious file artifacts
+  let MaliciousFileArtifacts = dynamic (["lsass.dmp","test.pwd","lsremora.dll","lsremora64.dll","fgexec.exe","pwdump","kirbi","wce_ccache","wce_krbtkts","wceaux.dll","PwHashes","SAM.out","SECURITY.out","SYSTEM.out","NTDS.out" "DumpExt.dll","DumpSvc.exe","cachedump64.exe","cachedump.exe","pstgdump.exe","servpw64.exe","servpw.exe","pwdump.exe","fgdump-log"]);
+  Event
+  | where EventLog == "Microsoft-Windows-Sysmon/Operational" and EventID==11
+  | parse EventData with * 'TargetFilename">' TargetFilename "<" *
+  | where TargetFilename has_any (MaliciousFileArtifacts)
+  | parse EventData with * 'ProcessGuid">' ProcessGuid "<" * 'Image">' Image "<" *
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated) by EventID, Computer, Image, ProcessGuid, TargetFilename
+entityMappings:
+- entityType: Account
+  fieldMappings:
+    - identifier: FullName
+      columnName: SubjectAccount
```

Comment:
SubjectAccount does not seem like is being brought in the result. However, it seems we are using it in the Entity mapping. Can you please double check this. Thanks.

Reply:
Changed